webapp/app/components/pages/login/LoginPage.css

```diff
@@ -0,0 +1,27 @@
+.card-container.card {
+    max-width: 350px;
+    padding: 40px 40px;
```

Comment:
if the outer padding is the same value (vertical == horizontal), 
there would be no need  to specify them separately.

`padding: [vertical] [horizontal];` 

rather please use 

`padding: 40px;`